src/models/ModelInterface.jl

```diff
@@ -601,7 +601,7 @@ end
 Future work: There is some subtlety in how accessor functions should be handled.
 TODO: The new instance methods do not yet handle the `context` keyword argument.
 """
-macro migrate(head)
+macro migrate_theory(head)
```

Comment:
Thanks Kevin!

This name feels off to me because we aren't migrating a theory, we're migrating a model along a theory. So I would suggest `@migrate_model`. @olynch, what do you think?

Reply:
Yeah, I was thinking something like "migrate, but for theories" but I agree that `migrate_model` seems like a better fit.

Reply:
`@migrate_model` sounds good to me!

Reply:
I changed it.